src/executor/vm/tests.rs

```diff
@@ -76,3 +76,19 @@ fn out_of_memory_access() {
     let module2 = super::Module::new(input, super::ExecHint::Untrusted).unwrap();
     assert!(super::VirtualMachinePrototype::new(&module2, |_, _, _| Ok(0)).is_err());
 }
+
+#[test]
+fn unsupported_type() {
+    let input = [
+        0x00, 0x61, 0x73, 0x6d, 0x01, 0x00, 0x00, 0x00, 0x01, 0x05, 0x01, 0x60, 0x00, 0x01, 0x7b,
```

Comment:
how this was generated? if something changes in VM, is there a script to regenerate it? or we'll have to update all the tests?

Reply:
It was provided to us in a bug report